src/include/platform/ConfigurationManager.h

```diff
@@ -192,6 +197,9 @@ class ConfigurationManager
     ConfigurationManager(const ConfigurationManager &)  = delete;
     ConfigurationManager(const ConfigurationManager &&) = delete;
     ConfigurationManager & operator=(const ConfigurationManager &) = delete;
+
+    uint16_t gSecuredPort   = CHIP_PORT;
```

Comment:
configurationmanager seems to have no data for any  other settings and it looks like a CRTP that various platforms implement.

This is not the right place to add a temporary in-memory config.

Reply:
moved port config to Mdns and Server